tests/library/trace-viewer.spec.ts

```diff
@@ -470,6 +470,22 @@ test('should show snapshot URL', async ({ page, runAndTrace, server }) => {
   await expect(traceViewer.page.locator('.browser-frame-address-bar')).toHaveText(server.EMPTY_PAGE);
 });
 
+test('should show copy button on snapshot url hover and copy to clipboard on click', async ({ page, runAndTrace, server }) => {
```

Comment:
This test fully includes the previous one. Let's merge them, keeping the old test name.